khronicle-core/src/commonTest/kotlin/FilterLevelLoggerTests.kt

```diff
@@ -9,287 +11,287 @@ class FilterLevelLoggerTests {
     fun filter_withMinimumLevelVerbose_allowsVerboseLogs() {
         val inner = CallListLogger()
         val outer = inner.withMinimumLogLevel(LogLevel.Verbose)
-        outer.verbose("tag", "message", Metadata(), null)
+        outer.verbose("tag", "message", buildMetadata(), null)
```

Comment:
Most of these didn't need to change to use `buildMetadata()` (the `Metadata` constructor would've worked fine), right?
Did you just do it for consistency?

Reply:
Gotcha, makes sense.